src/NexusMods.App.UI/Pages/MyLoadouts/GameLoadoutsSectionEntry/GameLoadoutsSectionEntryViewModel.cs

```diff
@@ -1,15 +1,130 @@
+            .Subscribe()
+            .DisposeWith(_compositeDisposable);
+
+        var createNewLoadoutCard = new CreateNewLoadoutCardViewModel()
+        {
+            AddLoadoutCommand = ReactiveCommand.CreateFromTask(async () =>
+            {
+                await _gameInstallation.GetGame().Synchronizer.CreateLoadout(_gameInstallation);
+            }),
+        };
+
+        _cardViewModelsSourceList.Add(createNewLoadoutCard);
+
+        _cardViewModelsSourceList.Connect()
+            .Merge(_loadoutCardViewModels.ToObservableChangeSet())
+            .Sort(new CardViewModelComparer())
+            .OnUI()
+            .Bind(out _cardViewModels)
+            .SubscribeWithErrorLogging()
+            .DisposeWith(_compositeDisposable);
+    }
+
+    private class CardViewModelComparer : IComparer<IViewModelInterface>
+    {
+        public int Compare(IViewModelInterface? x, IViewModelInterface? y)
+        {
+            if (x == null) return y == null ? 0 : -1;
+            if (y == null) return 1;
+
+            // Prioritize CreateNewLoadoutCardViewModel to always come first.
+            if (x is CreateNewLoadoutCardViewModel) return -1;
+            if (y is CreateNewLoadoutCardViewModel) return 1;
+
+            return (x, y) switch
+            {
+                (LoadoutCardViewModel lx, LoadoutCardViewModel ly) => string.Compare(lx.LoadoutName, ly.LoadoutName, StringComparison.Ordinal),
+                (SkeletonLoadoutCardViewModel sx, SkeletonLoadoutCardViewModel sy) => string.Compare(sx.LoadoutName, sy.LoadoutName, StringComparison.Ordinal),
+                (LoadoutCardViewModel lxs, SkeletonLoadoutCardViewModel sys) => string.Compare(lxs.LoadoutName, sys.LoadoutName, StringComparison.Ordinal),
+                (SkeletonLoadoutCardViewModel sxs, LoadoutCardViewModel lys) => string.Compare(sxs.LoadoutName, lys.LoadoutName, StringComparison.Ordinal),
+                _ => 0,
+            };
+        }
+    }
+
+
+    private void NavigateToLoadout(Loadout.ReadOnly loadout)
+    {
+        var loadoutId = loadout.LoadoutId;
+        Dispatcher.UIThread.Invoke(() =>
+            {
+                var workspaceController = _windowManager.ActiveWorkspaceController;
+
+                workspaceController.ChangeOrCreateWorkspaceByContext(
+                    context => context.LoadoutId == loadoutId,
+                    () => new PageData
+                    {
+                        FactoryId = LoadoutGridPageFactory.StaticId,
+                        Context = new LoadoutGridContext
+                        {
+                            LoadoutId = loadoutId,
+                        },
+                    },
+                    () => new LoadoutContext
+                    {
+                        LoadoutId = loadoutId,
+                    }
+                );
+            }
+        );
+    }
+
+    public void Dispose()
+    {
+        _compositeDisposable.Dispose();
     }
```

Comment:
Unused and should be removed.

Reply:
Right, this was supposed to be a service, but I ended up creating it manually. I guess I could call dispose in MyLoaoduts VM

Reply:
Updated the code to use this with `DisposeMany()` in `MyLoadoutsViewModel.cs`